src/app/dataset-view/additional-components/data-component/load-more/load-more.component.html

```diff
@@ -0,0 +1,32 @@
+<button mat-button
+        class="mr-4"
+        [disabled]="isAllDataLoaded"
```

Comment:
Maybe we should hide `<app-load-more>` block completely when all the data is loaded? I don't see value in having `Options` and related toggle group enabled, if wev'e done loading data. We could just hide the component entirely when the data is over.

Reply:
Agree on the first one.
But I think in the second one it's better to show somehow that this is all data,
I can change buttons to simple text, just to notify the user.

Reply:
I left the disabled button but I added a description, I think it's more understandable and user-friendly